packages/web-components/fast-foundation/src/anchor/anchor.template.ts

```diff
@@ -8,43 +8,45 @@ import type { Anchor } from "./anchor";
  * @public
  */
 export const AnchorTemplate: ViewTemplate<Anchor> = html`
-    <a
-        class="control"
-        part="control"
-        download="${x => x.download}"
-        href="${x => x.href}"
-        hreflang="${x => x.hreflang}"
-        ping="${x => x.ping}"
-        referrerpolicy="${x => x.referrerpolicy}"
-        rel="${x => x.rel}"
-        target="${x => x.target}"
-        type="${x => x.type}"
-        aria-atomic="${x => x.ariaAtomic}"
-        aria-busy="${x => x.ariaBusy}"
-        aria-controls="${x => x.ariaControls}"
-        aria-current="${x => x.ariaCurrent}"
-        aria-describedBy="${x => x.ariaDescribedby}"
-        aria-details="${x => x.ariaDetails}"
-        aria-disabled="${x => x.ariaDisabled}"
-        aria-errormessage="${x => x.ariaErrormessage}"
-        aria-expanded="${x => x.ariaExpanded}"
-        aria-flowto="${x => x.ariaFlowto}"
-        aria-haspopup="${x => x.ariaHaspopup}"
-        aria-hidden="${x => x.ariaHidden}"
-        aria-invalid="${x => x.ariaInvalid}"
-        aria-keyshortcuts="${x => x.ariaKeyshortcuts}"
-        aria-label="${x => x.ariaLabel}"
-        aria-labelledby="${x => x.ariaLabelledby}"
-        aria-live="${x => x.ariaLive}"
-        aria-owns="${x => x.ariaOwns}"
-        aria-relevant="${x => x.ariaRelevant}"
-        aria-roledescription="${x => x.ariaRoledescription}"
-        ${ref("control")}
-    >
-        ${startTemplate}
-        <span class="content" part="content">
-            <slot ${slotted("defaultSlottedContent")}></slot>
-        </span>
-        ${endTemplate}
-    </a>
+    <template @focus="${x => x.handleUnsupportedDelegatesFocus()}">
```

Comment:
If we added the event listener with code in the component's connected handler I think we could avoid needing to mess with the template at all.